Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM node:alpine
+FROM andthensome/alpine-node-bash
```

Comment:
Ah makes sense. I looked at the docs for Alpine linux and it seems you can install bash by putting `RUN apk add bash bash-doc bash-completion` in the dockerfile if we want to stick to the official images. It's up to you though.